en/resources/tutorials/tutorial_todo.md

```diff
@@ -55,8 +55,8 @@ import PackageDescription
 let package = Package(
     name: "TodoList",
     dependencies: [
-      .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 0),
-      .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git",  majorVersion: 0, minor: 16)
+      .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextVersion(from: "1.0.0")),
```

Comment:
Should be 2.0.0, again we should check this works.

Reply:
done